2021 01 19 shaun liu pim (#11399)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/unifiedrolemanagementpolicyruletarget.md

---
title: "unifiedRoleManagementPolicyRuleTarget resource type"
description: "A unifiedRoleManagementPolicyRuleTarget specifies the target associated with the role management policy."
author: "shauliu"
localization_priority: Normal
ms.prod: "microsoft-identity-platform"
doc_type: resourcePageType
---

# unifiedRoleManagementPolicyRuleTarget resource type

> Namespace: microsoft.graph

Reviewer: Add: [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

A unifiedRoleManagementPolicyRuleTarget specifies the target associated with the role management policy.
<!--
## Properties
|Property|Type|Description|
|:---|:---|:---|

> |caller|String|The caller for the policy rule target. One of None, Admin, EndUser.	|

Reviewer: ```suggestion
|caller|String|The caller for the policy rule target. Possible values: `None`, `Admin`, and `EndUser`. |
```

> |enforcedSettings|String collection|The list of settings which are enforced and cannot be overridden by child scopes. Use All for all settings.|

Reviewer: ```suggestion
|enforcedSettings|String collection|The list of settings that are enforced and can't be overridden by child scopes. Use `All` for all settings.|
```

> |inheritableSettings|String collection|The list of settings which can be inherited by child scopes. Use All for all settings.|

Reviewer: ```suggestion
|inheritableSettings|String collection|The list of settings that can be inherited by child scopes. Use `All` for all settings.|
```

> |level|String|The level for the policy rule target. One of Eligibility, Assignment.	|

Reviewer: ```suggestion
|level|String|The level for the policy rule target. Possible values: `Eligibility`, `Assignment`. |
```

> |operations|String collection|The operations for policy rule target. One of All, Activate, Deactivate, Assign, Update, Remove, Extend, Renew.|

Reviewer: ```suggestion
|operations|String collection|The operations for policy rule target. Possible values: `All`, `Activate`, `Deactivate`, `Assign`, `Update`, `Remove`, `Extend`, `Renew`.|
```

## Relationships
|Relationship|Type|Description|
|:---|:---|:---|

> |targetObjects|[directoryObject](../resources/directoryobject.md) collection|The collection of users, groups and servicePrincipals which are in scope of the policy. If not specified, all objects are in scope of the policy.|

Reviewer: ```suggestion
|targetObjects|[directoryObject](../resources/directoryobject.md) collection|The collection of users, groups and servicePrincipals that are in scope of the policy. If not specified, all objects are in scope of the policy.|
```
